save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/s3/DefaultS3Operations.kt

```diff
@@ -25,8 +33,13 @@ import java.util.concurrent.*
  */
 class DefaultS3Operations(
     properties: S3OperationsProperties,
-) : S3Operations {
+) : S3Operations, Closeable {
```

Comment:
Probably, it's better to use `AutoCloseable` here (almost the same signature, more modern API).

Reply:
sure